app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
```

Comment:
I think this class is redundant because we can add before_action permit params in application controller like doc github of this gem